test/network.bats

```diff
@@ -34,20 +46,29 @@ load helpers
 	echo "$output"
 	[ "$status" -eq 0 ]
 	pod1_id="$output"
+	run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod1_id"
+	echo "$output"
+	[ "$status" -eq 0  ]
+	ctr1_id="$output"
 
 	temp_sandbox_conf cni_test
 
 	run crioctl pod run --config "$TESTDIR"/sandbox_config_cni_test.json
 	echo "$output"
 	[ "$status" -eq 0 ]
 	pod2_id="$output"
+	run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod2_id"
+	echo "$output"
+	[ "$status" -eq 0  ]
+	ctr2_id="$output"
 
-	ping_pod_from_pod $pod1_id $pod2_id
+	ping_pod_from_pod $ctr1_id $ctr2_id
```

Comment:
Any value to keeping the test for pod1 to pod2 in play and adding the ctr1 to ctr2 test in addition?